include/ddprof_input.hpp

```diff
@@ -97,7 +97,8 @@ typedef struct DDProfInput {
   XX(DD_PROFILING_NATIVE_GLOBAL,        global,             g, 'g', 1, input, NULL, "", )                      \
   XX(DD_PROFILING_INTERNAL_STATS,       internal_stats,     b, 'b', 1, input, NULL, "", )                      \
   XX(DD_PROFILING_NATIVE_SOCKET,        socket,             Z, 'Z', 1, input, NULL, "", )                      \
-  XX(DD_PROFILING_NATIVE_PRESET,        preset,             D, 'D', 1, input, NULL, "", )
+  XX(DD_PROFILING_NATIVE_PRESET,        preset,             D, 'D', 1, input, NULL, "", ) \
+  XX(DD_PROFILING_NATIVE_SHOW_SAMPLES,  show_samples,       Y, 'Y', 0, input, NULL, "", )
```

Comment:
We're running out of single-character short options :)

This is a gray area, but capital letters elsewhere designate options that affect the profiler's interaction with Datadog's backend (metadata) whereas lowercase characters designate options that affect the profiler's behavior itself (incl. logging).  This is a slightly silly convention--do we want to keep it?

Reply:
Sorry I was not aware of that convention !
I will make the change.
We are definitely running out of characters, it would be nice to have only long format for some options, but it seems difficult with the current design.

Reply:
Well, the convention isn't really documented anywhere and doesn't even make that much sense.

I was asking a genuine question--does that convention even make sense?  Maybe we should just get rid of this longopt nonsense and do proper input handling in a future PR?